internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -507,7 +497,7 @@ func (opts *Opts) newClusterName() string {
 	i := 0
 	if clusters, ok := cs.([]atlas.Cluster); ok {
 		for {
-			clusterName := "QuickstartCluster" + strconv.Itoa(i)
+			clusterName := autogeneratedName()
```

Comment:
I think we can remove the  the loop and API call right?

Reply:
yes, correct